### PR TITLE
Harmful initialization of testing point.

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Related to #1548.

### CODE REVIEW

File: kratos/tests/geometries/test_triangle_2d_3.cpp [hunk 1]
```diff
@@ -205,10 +205,10 @@ namespace Testing {
     auto geom = GenerateIrregularTriangle2D3<Node<3>>();
```
Comment (roigcarlo) [inline kratos/tests/geometries/test_triangle_2d_3.cpp]: I believe that @pooyan-dadvand was asking for this one to use points?

Comment (rubenzorrilla) [reply kratos/tests/geometries/test_triangle_2d_3.cpp]: Yes, but having a look to that I found this minor issue.

**Comment by pooyan-dadvand** [comment]
It would be better to create directly a point (which is derived from array_1d) and use it like a vector:

```c++
   Point baricentre = geom_pts[0] + geom_pts[1] + geom_pts[2];
   baricentre *= 1.00 / 3.00;
```

**Review by RiccardoRossi — DISMISSED** [review]

**Comment by rubenzorrilla** [comment]
@pooyan-dadvand done!

**Review by pooyan-dadvand — CHANGES_REQUESTED** [review]
As @roigcarlo mentioned the `GenerateIrregularTriangle2D3` is passing wrong arguments to constructor if point type is a node (The first argument should be Id of the node) so I would suggest to have two different methods to create a triangle of node and a triangle of point

**Comment by rubenzorrilla** [comment]
For me it's the best approach as well. I will implement that.

**Comment by rubenzorrilla** [comment]
@pooyan-dadvand done!

**Review by pooyan-dadvand — APPROVED** [review]
:+1:

**Pull request merged**